README.md

```diff
@@ -1,2 +1,17 @@
 # web
 Web app for checking tmt tests, plans and stories
+# API
+API for checking tmt tests and plans metadata
+* `/`
+  * `test-url` - URL of the repo test is located in - accepts a `string`
+  * `test-name` - Name of the test - accepts a `string`
+  * `test-ref` - Ref of the repository the test is located in - accepts a `string`, default is `main`
```

Comment:
The [fmf id](https://fmf.readthedocs.io/en/stable/concept.html#identifiers) defines also `path` which points to the directory where the `fmf` tree is stored. Please, include it in test/plan parameters as well.

Reply:
Here's a [real example](https://github.com/beakerlib/beakerlib/pull/171/files) of a remote plan referencing.

Reply:
Added parameter for `path`

Reply:
Thanks. Just a nitpick: Let's keep the keys in logical order, e.g. `url`, `ref`, `path`, `name`.